libsofia-sip-ua/nta/nta.c

```diff
@@ -4634,6 +4646,15 @@ char const *nta_leg_get_rtag(nta_leg_t const *leg)
     return NULL;
 }
 
+/** Remove remote tag. */
+void nta_leg_del_rtag(nta_leg_t *leg)
+{
+  if (leg && leg->leg_remote) {
+    sip_to_t to0[1]; *to0 = *leg->leg_remote; to0->a_params = NULL;
```

Comment:
I'd put these on separate lines:
sip_to_t to0[1]; 
*to0 = *leg->leg_remote; to0->a_params = NULL;

but it's just a preference of mine.